testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/maven/content/MavenContentValidationIT.java

```diff
@@ -61,11 +59,7 @@
   @org.ops4j.pax.exam.Configuration
   public static Option[] configureNexus() {
     return options(nexusDistribution("org.sonatype.nexus.assemblies", "nexus-base-template"),
-        // TODO: This should be replaced with:
-        // mavenBundle("org.sonatype.http-testing-harness", "server-provider").versionAsInProject()
-        // ..once the http-testing-harness duplicate import of org.sonatype.tests.http.server.api is corrected.
-        wrappedBundle(maven("org.sonatype.http-testing-harness", "server-provider").versionAsInProject())
-            .overwriteManifest(OverwriteMode.FULL).instructions("DynamicImport-Package=*")
+        mavenBundle(maven("org.sonatype.http-testing-harness", "server-provider").versionAsInProject())
```

Comment:
Could be simplified further to:
`mavenBundle("org.sonatype.http-testing-harness", "server-provider").versionAsInProject()`

since the `maven(groupId, artifactId)` element is only needed when wrapping bundles... but it also doesn't do any harm, so if you prefer it for consistency's sake then leave it as is :)